src/imitation/scripts/config/train_imitation.py

```diff
@@ -96,7 +97,14 @@ def seals_humanoid():
     environment = dict(gym_id="seals/Humanoid-v0")
 
 
+@train_imitation_ex.named_config
+def dqn():
```

Comment:
Is this used for anything other than sqil? If not, does it need to live in this file? (I'm particularly confused that it is changing something in `rl` but `train_imitation` does not directly depend on the `rl` ingredient.)